libmscore/rendermidi.cpp

```diff
@@ -154,13 +155,21 @@ void Score::updateChannel()
 //---------------------------------------------------------
 
 static void playNote(EventMap* events, const Note* note, int channel, int pitch,
-   int velo, int onTime, int offTime)
+   int velo, int onTime, int offTime, const Scale& s)
       {
       if (!note->play())
             return;
       velo = note->customizeVelocity(velo);
+
+      double scaleTuning = s.getTuning(note);
+      if (fabs(scaleTuning) > 200) {
+            int step = (int)scaleTuning / 100;
+            pitch += step;
+            scaleTuning += step * 100;
```

Comment:
Also, I know to be almost alone in this, but I find literal numeric values reader-hostile. I would replace `100` with a symbolic value, say `centPerSemitone`, defined at the top and `200` with `2 * centPreSemitone`.

But, I know, these out-of-fashion concerns, nowadays...